openlibrary/components/ObservationForm/components/OLChip.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <span
+    class="chip"
+    v-bind:class="[{ 'chip--selectable': selectable, 'chip--selected': isSelected }, classList]"
+    @[canSelect]="onClick"
+    >
+    <slot name="before"></slot>
+    {{text}}
+    <slot name="after"></slot>
+  </span>
+</template>
+
+<script>
+/**
+ * Displays text within a pill-shaped outline.
+ */
+export default {
+    name: 'OLChip',
```

Comment:
We can call this a pill if it makes more sense to everyone.  Prefaced this with "OL" because: (a) Component names should have more than one word, and (b) It is generic enough to be used in other OL Vue components.